code/crates/core-consensus/src/types.rs

```diff
@@ -114,3 +115,9 @@ pub enum VoteExtensionError {
     #[error("Invalid vote extension")]
     InvalidVoteExtension,
 }
+
+#[derive_where(Clone, Debug, PartialEq, Eq)]
+pub enum GossipMsg<Ctx: Context> {
```

Comment:
I am a bit wary of calling those "gossip" messages and of naming the channel "/gossip" as it might be very confusing for users. How about renaming those to "liveness" instead? It's not great either, and less correct than "gossip" but at least it avoids the confusion. What do you think?

Reply:
I am ok with it. @nenadmilosevic95 what do you think?

Reply:
2526b2c017db83ed680f800c59a6efea783246d1